flow/raster_cache.h

```diff
@@ -42,7 +42,15 @@ struct PrerollContext;
 
 class RasterCache {
  public:
-  explicit RasterCache(size_t threshold = 3);
+  // The default max number of picture raster caches to be generated per frame.
+  // Generating too many caches in one frame may cause jank on that frame. This
+  // limit allows us to throttle the cache and distribute the work across
+  // multiple frames.
+  static constexpr int kDefaultPictureCacheLimitPerFrame = 3;
+
+  explicit RasterCache(
+      size_t threshold = 3,
```

Comment:
Maybe we should rename this to `access_threshold` now. In a later patch is fine.

Reply:
Ack.